docs/manual/scala/guide/cluster/PersistentEntityRDBMS.md

```diff
@@ -61,3 +61,16 @@ The database schemas needed for the tables can be found [here](https://github.co
 The full configuration options that Lagom provides for managing the creation of tables is here:
 
 @[persistence](../../../../../persistence-jdbc/core/src/main/resources/reference.conf)
+
+## Application Loader
+
+Your Lagom Application Loader will by default load the Cassandra persistence components. For an RDBMS not only the JDBC persistence components must be loaded but also a connection pool must be configured. By default the Hikari connection pool is available in Play and can be configured as follows:
+
+```
+abstract class BlogApplication(context: LagomApplicationContext)
+  extends LagomApplication(context)
+    with JdbcPersistenceComponents
+    with HikariCPComponents
+    with AhcWSComponents {
+```
+
```

Comment:
Lagom docs try to avoid adding embedded code since that's simple text that can't be compiled on CI. Instead, code is embedded on a preprocessing stage after code compilation was run (in some cases, code is not only compiled but also executed). See how [ReadSideCassandra.md] imports two snippets: 

```
@[imports](code/docs/home/scaladsl/persistence/CassandraReadSideQuery.scala)
@[service-impl](code/docs/home/scaladsl/persistence/CassandraReadSideQuery.scala)
```
from source code: https://github.com/lagom/lagom/blob/master/docs/manual/scala/guide/cluster/code/docs/home/scaladsl/persistence/CassandraReadSideQuery.scala#L3-L11 and https://github.com/lagom/lagom/blob/master/docs/manual/scala/guide/cluster/code/docs/home/scaladsl/persistence/CassandraReadSideQuery.scala#L21-L31
And the final result: http://www.lagomframework.com/documentation/1.3.x/scala/ReadSideCassandra.html


There are few details to note here:

 * `code/` folder is inside the `manual/scala/guide/cluster/``
 * all scala code for page `ReadSideCassandra.md` of the docs is on `CassandraReadSideQuery.scala` (this is probably not a very good example since the amount of code is small, but see https://github.com/lagom/lagom/blob/master/docs/manual/scala/guide/services/code/ServiceDescriptors.scala for a single `scala` file with all snippets of a single `markdown` page.
 * a snippet is delimited with `//#snippet-id`
 * See https://github.com/lagom/lagom/blob/master/docs/README.md for tips on running a local copy of the docs and also validating, compiling and testing the code embedded.